### DISCUSSION TITLE
advised to add a feature

### DISCUSSION THREAD

**Discussion by Nyx2022** [projectdiscovery/rawhttp#48] [General]
in python,we can set host in headers to the value that different from the real host of http url,however,by my test[rawhttp](https://github.com/projectdiscovery/rawhttp)) may not support this feature,so i advise [rawhttp](https://github.com/projectdiscovery/rawhttp)to support this feature

**Comment by Mzack9999** [comment]
@Nyx2022 The library already allows you to set the host header by providing it within the headers map. Feel free to post how you use the library if you need any suggestions.
